music_assistant/server/providers/jellyfin/const.py

```diff
@@ -70,3 +73,7 @@
 
 USER_APP_NAME: Final = "Music Assistant"
 USER_AGENT: Final = "Music-Assistant-1.0"
+
+UNKNOWN_ARTIST_MAPPING = ItemMapping(
+    media_type=MediaType.ARTIST, item_id="[unknown]", provider=DOMAIN, name="[unknown]"
```

Comment:
I used '[unknown]' to match musicbrainz. I could use the UNKNOWN_ARTIST constant instead, and i could also update that constant to match musicbrainz.

Reply:
If you see improvements in the core/base implementation, always feel free to adjust in your PR or a separate follow-up one. I think it makes sense if we have a global constant value for an unknown artist, including musicbrainz id

Reply:
Have added the MBID to core, but core has this:

`UNKNOWN_ARTIST: Final[str] = "Unknown Artist"`

So wondered whether to prefer that, or what musicbrainz used.

Reply:
I think we should follow musicbrainz

Reply:
Done - '`music_assistant.server.constants.UNKNOWN_ARTIST`' now `'[unknown]`'